backup_and_restore/application_backup_and_restore/migration-debugging-velero-admission-webhooks.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
```

Comment:
This was part of an experiment -- I wanted to nest assemblies, not realizing that is not allowed in openshift (it will be OK for Jupiter, btw). I deleted this file.